### PR TITLE
Prettify output

### PR DESCRIPTION
**Pull request by typhoonzero**
Fix https://github.com/PaddlePaddle/cloud/issues/195

Note here's still a problem when train exited with error, submit with:

```
-entry "sleep 30 && exit 136"
```
will not show message when `paddlecloud get workers [jobname]`

### CODE REVIEW

File: docker/paddle_k8s [hunk 1]
```diff
@@ -11,6 +11,23 @@ start_pserver() {
...
+  echo "job returned $ret...setting pod return message..."
+  echo "==============================="
+  if [ $ret -eq 136 ] ; then
+    echo "Error Arithmetic Operation(Floating Point Exception)" > /dev/termination-log
+  elif [ $ret -eq 139 ] ; then
+    echo "Segmentation Fault" > /dev/termination-log
+  elif [ $ret -eq 1 ] ; then
+    echo "General Error" > /dev/termination-log
+  elif [ $ret -eq 134 ] ; then
+    echo "Program Abort" > /dev/termination-log
```
Comment (Yancey1989) [inline docker/paddle_k8s]: Deal with an unknown return code?

File: go/paddlecloud/get.go [hunk 2]
```diff
@@ -122,34 +132,71 @@ func registry() error {
...
 	err = json.Unmarshal(respBody, &respObj)
 	if err != nil {
 		return err
 	}
 	items := respObj.(map[string]interface{})["items"].([]interface{})
+
 	w := tabwriter.NewWriter(os.Stdout, 0, 0, 3, ' ', 0)
 	if len(items) >= 0 {
-		fmt.Fprintf(w, "NUM\tNAME\tSUCC\tFAIL\tSTART\tCOMP\tACTIVE\t\n")
+		fmt.Fprintf(w, "NAME\tACTIVE\tSUCC\tFAIL\tSTART\tCOMP\tPS_NAME\tPS_READY\tPS_TOTAL\t\n")
```
Comment (Yancey1989) [inline go/paddlecloud/get.go]: What's the PS_NAME used for? Maybe only print the stats of PServer?

File: docker/paddle_k8s (hunk 1 above)
Comment (typhoonzero) [inline docker/paddle_k8s]: unknown就不现实message了，ret code还是可以看到的

File: go/paddlecloud/get.go (hunk 2 above)
Comment (typhoonzero) [inline go/paddlecloud/get.go]: replicaset的name不过对于用户来说没什么用了倒是。可以去掉

**Pull request merged**